src/App.tsx

```diff
@@ -1,10 +1,42 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, filterByStatus } from './api/todos';
+import { typeTodo } from './types/Todo';
+import { Todo } from './components/Todo/Todo';
+import { Loader } from './components/Loader/Loader';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterType, setFilterType] = useState('all');
+  const [todos, setTodos] = useState<typeTodo[]>([]);
+
+  const filterStatus = (filter: string) => {
+    let status: string | boolean = '';
```

Comment:
no need to set a boolean value, just handle different string values. But again - it's better to create enum for it